internal/printer/printer.go

```diff
@@ -221,7 +226,7 @@ func (p *printer) printStyleOrScript(n *astro.Node) {
 }
 
 func (p *printer) printAttribute(attr astro.Attribute) {
-	if attr.Key == "define:vars" {
+	if attr.Key == "define:vars" || attr.Key == "set:text" || attr.Key == "set:html" {
 		return
```

Comment:
If one of these slips through (possible if there are duplicates or if `set:html` and `set:text` are mixed), we ignore them and do not print them as attributes.